inc/notificationtargetcommonitilobject.class.php

```diff
@@ -1010,9 +1010,7 @@ function getDataForObject(CommonDBTM $item, array $options, $simple = false) {
             );
          }
 
-         $data["##$objettype.solution.description##"] = Toolbox::unclean_cross_side_scripting_deep(
-            $itilsolution->getField('content')
-         );
+         $data["##$objettype.solution.description##"] = $itilsolution->getField('content');
```

Comment:
A little doubt about this one

Reply:
This is useless as it is done by the `Html::entity_decode_deep($data);` call on https://github.com/glpi-project/glpi/pull/4401/files#diff-4ab777b773d4ae9eeccdbc10ca255a3cR275.